Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -387,6 +387,12 @@ static bool is_forbidden_method(String const& method)
     return lowercase_method.is_one_of("connect", "trace", "track");
 }
 
+static bool is_method(String const& method)
```

Comment:
According to https://datatracker.ietf.org/doc/html/rfc7231#section-8.1, the registry for all available methods is much larger than this.
See: https://www.iana.org/assignments/http-methods/http-methods.xhtml

Reply:
This is solved by following the solution suggested here: https://github.com/SerenityOS/serenity/pull/13476#discussion_r841597657